lib/src/sdk_env.dart

```diff
@@ -500,3 +508,13 @@ String _join(String path, String binDir, String executable) {
   }
   return cmd;
 }
+
+/// Thrown when `dartfmt` fails to run.
+class DartFmtFailed implements Exception {
```

Comment:
Let's call this `ToolException` so we can reuse it across tools?

Reply:
Love it!